### DISCUSSION TITLE
Nightly Builds

### DISCUSSION THREAD

**Discussion by alabotski** [winsiderss/systeminformer#1470] [Questions / Answers] [answered]
Will support for nightly builds be or will they be discontinued as well?
https://processhacker.sourceforge.io/nightly.php

**Comment by dmex** [comment]
Nightly builds will return after migration has completed

**Comment by Biswa96** [comment]
Some nightly builds pop up for a split of second in this repository

**Reply by dmex** [reply]
We're testing the release pipeline and they're not ready for the public yet... Don't use them otherwise don't blame us when your installation is destroyed or your machine catches fire.

**Reply by Biswa96** [reply]
Just curious. After SI automatically updates, the timestamp of ksi.dll does not change. Isn't ksi.dll updated/replaced at install time?

**Reply by jxy-s** [reply]
This is on purpose. We overwrite `ksi.dll` only if it needs updated. If the hash in the update is the same as what is on disk, it isn't overwritten.

**Comment by dmex** [comment, marked as answer]
Nightly builds are live 🥳

https://systeminformer.sourceforge.io/nightly.php

**Reply by yossizahn** [reply]
Great news, thanks!

Has there been progress on ARM64 builds?

**Reply by DavidXanatos** [reply]
Congratulations to the new nightlies, very cool to see good progress...

**Comment by alabotski** [comment]
Hi 
I asked @dmex to add RSS support for nightly builds (https://github.com/winsiderss/systeminformer/issues/910)
Is it possible to add this functionality to a new version (nightly builds) of the program?

**Reply by dmex** [reply]
Github provides RSS:
https://github.com/winsiderss/si-builds/releases.atom
